lib/checks/label/applet-alt-alt.js

```diff
@@ -0,0 +1,4 @@
+let label = node.getAttribute('alt');
```

Comment:
This check shouldn't look at the content, you're already doing this with `none-empty-text`. I also think this check should be renamed to `none-empty-alt` or something like it. There is nothing specific to applets in this check to warrant putting `applet` in the check ID.

Reply:
The reason I am looking at content in this context is because as you have stated, VoiceOver do not detect the alt on the applet tag. And as mentioned in the docs for H35(https://www.w3.org/TR/WCAG20-TECHS/H35.html), the support of the alt tag is varying in browsers. So there needs to be text inside the applet tag to have visible text. With area label or labeledby this is not needed.

And if you have any suggestions to check this differently I am all ears. I did forget to test with an applet loaded into the applet tag, and will do this as soon as possible :)